.github/ISSUE_TEMPLATE/web_app_vulnerability_scan.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Run the web app vulnerability scans
+about: Issue template for the monthly web app vulnerability scans and triaging of the findings
+title: Monthly web app vulnerability scans
+labels: +,compliance,infra,no demo,orange
+assignees: dsotirho-ucsc
```

Comment:
I know you've been the point person for the task, but consider confirming this. It may be easier to detect an unassigned issue than an assigned one.